src/client/components/body-portal/body-portal.tsx

```diff
@@ -44,6 +48,9 @@ export class BodyPortal extends React.Component<BodyPortalProps, {}> {
     super(props);
     this.target = document.createElement("div");
     this.target.className = classNames("body-portal", { "full-size": props.fullSize });
+    this.state = {
```

Comment:
I would go with `state` class property initialisation to be consistent. I know we have constructor for shenanigans with `target`, but keep it just for that.

Reply:
👍